### PR TITLE
Separate developer readme and public readme

### PR DESCRIPTION
**Pull request by andreybavt**
simplified readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,591 +1,51 @@
...
+    <img src="readme/inspect.png" alt="Administration panel" />
+</p>
 
-* [Docker](https://www.docker.com/)
-* [Docker Compose](https://docs.docker.com/compose/install/)
-* [Poetry](https://python-poetry.org/) for Python package and environment management
+<br/>
 
-## Frontend Requirements
+Giskard is an open-source quality assurance platform for AI models.
```
Comment (alexcombessie) [inline README.md:13]: Perhaps this block would be better positioned above the screenshot?

File: README.md [hunk 2]
```diff
@@ -1,591 +1,51 @@
...
 
-* Node.js (with `npm`).
+- **Inspect** python models using visual interface
```
Comment (alexcombessie) [inline README.md:15]: These 3 bullet points may need a bit of re-wording to make them more impactful. 

Ex: 
- **Inspect** ML models visually from your Python notebook 📗
- **Review** ML models collaboratively to get feedback on potential errors 🧐
- **Test** ML models automatically for a safe deployment to production ⛑

WDYT @jmsquare ?

**Review by alexcombessie — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (andreybavt) [reply README.md:13]: agree

File: README.md (hunk 2 above)
Comment (andreybavt) [reply README.md]: I like this version, I pushed it

**Pull request merged**